store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -66,9 +63,6 @@
     this.readers = readers;
     this.index = 0;
     this.currentReader = readers.get(0);
-    CarbonTaskInfo carbonTaskInfo = new CarbonTaskInfo();
```

Comment:
Whether will it affect CSDK?

Reply:
The same issue would be there in CSDK as well as this would be called internally.

Reply:
I am testing.

Reply:
ok

Reply:
I also can't reproduce in local, my laptop has 8 cores and add 
```
    CarbonProperties.getInstance().addProperty("enable.unsafe.in.query.processing","true");```

 But SDK is run in single machine, so how to reproduce it？ 

Can you write the error in JIRA or this PR?

Reply:
@xubo245 i have update the JIRA. Please check

Reply:
so how to reproduce it？

Reply:
@xubo245 Please continue conversation on Jira.

Reply:
I tried the JIRA code in my laptop， JVM don‘'t crash...